internal/services/rate/privat.go

```diff
@@ -0,0 +1,62 @@
+package rate
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+const (
+	PrivatExchangeRateURL = "https://api.privatbank.ua/p24api/pubinfo?json&exchange&coursid=5"
```

Comment:
Are you inspired by Dmytro's PR?)

Reply:
What kind of inspiration?)

Reply:
You've used the same providers as Dmytro)